src/style.css

```
@@ -0,0 +1,124 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.container__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 3px 40px;
+}
+
+.container__headline {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 0 4px;
+}
+
+.container__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin: 0 0 16px;
+}
+
+.container__revies {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0 0  24px;
+}
+
+.container__count-review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.container__cost-info {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0  16px;
+
+}
```

the same recomendation
```suggestion
  margin-bottom: 16px;
}
```